tests/unit-tests/test-class-usage-tracking.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+class Sensei_Usage_Tracking_Test extends WP_UnitTestCase {
+	public function setup() {
+		parent::setup();
+		$this->factory = new Sensei_Factory();
```

Comment:
It doesn't look like you're testing course or lessons data, so you don't need to include `Sensei_Factory`.

Reply:
Removed